src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -857,10 +857,9 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
 
             QHash<QString, QString> extraVersionInfo;
 
-            mixxx::BeatsPointer pBeats = BeatFactory::makePreferredBeats(
-                    *track, beats, extraVersionInfo, false, false, sampleRate, 0, 0, 0);
-
-            track->setBeats(pBeats);
+            auto pBeats = new mixxx::BeatMap(*track, sampleRate, beats);
```

Comment:
I think we can remove the ```if (time < 1)``` check above. This is suspicious anyway, because if the beat is in the lead in, it is wrong to place it at 0,001 instead.